### PR TITLE
[WIP] Add sale_amendment

### PR DESCRIPTION
**Pull request by guewen**
**Work in Progress**

Depends on
- [x] https://github.com/odoo/odoo/pull/6036
- [x] ~~OCA/OCB#184~~
- [x] https://github.com/OCA/stock-logistics-workflow/pull/94

### CODE REVIEW

File: sale_amendment/view/sale_order.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+
+    <record id="view_order_form" model="ir.ui.view">
+      <field name="name">sale.order.form.amendment</field>
+      <field name="model">sale.order</field>
+      <field name="inherit_id" ref="sale_stock.view_order_form_inherit"/>
+      <field name="arch" type="xml">
+         <button name="ship_recreate">
```
Comment (gurneyalex) [inline sale_amendment/view/sale_order.xml]: relying on the default value of position... :frowning: explicit is better that explicit IMO (and I never remember that default value...)

Comment (yvaucher) [inline sale_amendment/view/sale_order.xml]: Default position is `inside` so there is a :bug: here

Comment (guewen) [inline sale_amendment/view/sale_order.xml]: Oops, it was not intentional. I would prefer an error than a default value in such situation.

**Comment by legalsylvain** [comment]
Hi @guewen,
Is this PR still in WIP ?
Regards.

**Comment by guewen** [comment]
@legalsylvain I need to check with @gurneyalex or @lepistone (both on holidays for now)

**Comment by Cedric-Pigeon** [comment]
Hi @gewen, I made some functionnal tests on this PR. It is impossible to amend a sale order which has an invoice. What about an advance invoice which has no link with sale order lines ?
What is your opinion ?

**Comment by guewen** [comment]
> Hi @gewen, I made some functionnal tests on this PR. It is impossible to amend a sale order which has an invoice. What about an advance invoice which has no link with sale order lines ?
> What is your opinion ?

Hi, thanks for your tests! I'm not sure this use case was meant to be supported. @jgrandguillaume may have more thought about that.

**Comment by Cedric-Pigeon** [comment]
Hi @guewen I made a PR on your original branch. Regards.

**Comment by pedrobaeza** [comment]
Is this going to be continued?

**Comment by pedrobaeza** [comment]
It seems no, as 9.0 includes amendment  in core.

**Pull request closed without merging**